cockatrice/src/dlg_update.cpp

```diff
@@ -142,8 +142,8 @@ void DlgUpdate::finishedUpdateCheck(bool needToUpdate, bool isCompatible, Releas
             tr("A new version of Cockatrice is available!") + "<br><br>"
             + "<b>" + tr("New version") + QString(":</b> %1<br>").arg(release->getName())
             + "<b>" + tr("Released") + QString(":</b> %1 <a href=\"%2\">(").arg(publishDate, release->getDescriptionUrl()) + tr("Changelog") + ")</a><br><br>"
-            + tr("Unfortunately there are no download packages available for your operating system. \nYou may have to build from source yourself.") + "<br><br>"
-            + tr("Please check the download page manually and visit the wiki for instructions on compiling."));
+            + tr("Unfortunately there could no download package for your operating system be found. Please check the download page manually.") + "<br><br>"
```

Comment:
???

Reply:
???

Reply:
`Unfortunately there could no download package for your operating system be found.` appears to be a typo or copy/paste error. Maybe change it to `No download package for your operating system could be found.`?

Reply:
https://github.com/Cockatrice/Cockatrice/pull/2718/commits/212af075d24e44af432a9e247a490dc2ec197162 should addresses this. What do you think @skwerlman ?

Are this the same complains you had @Nightfirecat ?
I wasn't sure what you try to point out.
Make sure to review/comment on the most recent code next time - and give me some hint, please. :P

Reply:
212af07 does indeed fix the issue i saw :+1:

Reply:
Yes, that does fix it. 👍

Reply:
Ok, so thanks for reporting back. And poking in the first place! ;)